2022-05-02 11:41:01,065 INFO  ==============================================================
2022-05-02 11:41:01,066 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-02 11:41:01,068 INFO  Dir:   software_management_end_to_end/
2022-05-02 11:41:01,068 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-02 11:41:01,069 INFO  ==============================================================
2022-05-02 11:41:01,079 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-02 11:41:01,080 INFO  EnvironmentC8y Setup
2022-05-02 11:41:01,693 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-02 11:41:01,798 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
2022-05-02 11:41:01,800 INFO  Delaying execution by 7 seconds
2022-05-02 11:41:22,770 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 13 secs
2022-05-02 11:41:23,392 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-02 11:41:23,856 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-02 11:41:24,319 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-02 11:41:26,542 INFO  State of current operation: SUCCESSFUL
2022-05-02 11:41:26,764 INFO  Response status: 201
2022-05-02 11:41:26,765 INFO  Response to action: {
    "creationTime": "2022-05-02T11:41:26.725Z",
    "deviceName": "m32sd10d",
    "deviceId": "18127040",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168196",
    "id": "18168196",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-02 11:41:26,766 INFO  Started operation: {'creationTime': '2022-05-02T11:41:26.725Z', 'deviceName': 'm32sd10d', 'deviceId': '18127040', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18168196', 'id': '18168196', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-02 11:41:28,982 INFO  State of operation 18168196 : PENDING
2022-05-02 11:41:31,194 INFO  State of operation 18168196 : EXECUTING
2022-05-02 11:41:33,411 INFO  State of operation 18168196 : EXECUTING
2022-05-02 11:41:35,627 INFO  State of operation 18168196 : EXECUTING
2022-05-02 11:41:37,858 INFO  State of operation 18168196 : EXECUTING
2022-05-02 11:41:40,076 INFO  State of operation 18168196 : FAILED
2022-05-02 11:41:40,077 ERROR Stopping as the operation has failed
2022-05-02 11:41:40,078 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-02 11:41:40,101 WARN  SystemError: The operation has failed ... blocked 
2022-05-02 11:41:40,102 INFO  
2022-05-02 11:41:40,103 INFO  cleanup:
2022-05-02 11:41:44,054 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-02 11:41:44,622 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-02 11:41:44,626 INFO  
2022-05-02 11:41:44,627 INFO  Test duration: 43.57 secs
2022-05-02 11:41:44,628 INFO  Test final outcome:  BLOCKED
2022-05-02 11:41:44,629 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-02 11:41:44,630 INFO  
